test_regress/t/t_dist_attributes_src.pl

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2022 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(dist => 1);
+my $root = "..";
+# some of the files are only used in verilation
+# and are only in "include" folder
+my $input_dirs = "$root/src";
+my $clang_args = "-I$root/src/ -I$root/include/ -I$root/include/vltstd/ -I$root/src/obj_opt/ -fcoroutines-ts";
```

Comment:
```suggestion
my $clang_args = "-I$root/src/ -I$root/include/ -I$root/src/obj_opt/ -fcoroutines-ts";
```

For source run vltstd not needed.